src/js/auth.js

```diff
@@ -148,9 +174,12 @@ function forceLogin(appState = '/') {
   loginPromptView.render();
 }
 
-export {
+Radio.reply('auth', {
```

Comment:
moving the exports to `Radio` prevents this file from being included in more places as now it's only included by the `import()`